### PR TITLE
Fix mon, osd, pg status wise counts

### PR DESCRIPTION
**Pull request by anmolbabu**
tendrl-bug-id: Tendrl/performance-monitoring#188
Signed-off-by: anmolbabu <anmolbudugutta@gmail.com>

### CODE REVIEW

**Comment by anmolbabu** [comment]
Verified

**Comment by anmolbabu** [comment]
@r0h4n @nthomas-redhat @shtripat Please review

File: tendrl/performance_monitoring/sds/ceph/ceph_plugin.py [hunk 1]
```diff
@@ -363,14 +362,16 @@ def get_mon_status_wise_counts(self, cluster_id):
...
+            mons = json.loads(mons.get('data', '{}'))
+            mons = mons['mons']
             mon_status_wise_counts['total'] = len(mons)
             outside_quorum = etcd_read_key(
-                '/clusters/%s/maps/mon_status/data/outside_quorum' % cluster_id
+                '/clusters/%s/maps/mon_status/data' % cluster_id
             )
-            outside_quorum = ast.literal_eval(outside_quorum['outside_quorum'])
+            outside_quorum = json.loads(outside_quorum.get('data', '{}'))
+            outside_quorum = outside_quorum['outside_quorum']
```
Comment (r0h4n) [inline tendrl/performance_monitoring/sds/ceph/ceph_plugin.py]: outside_quorum can be an empty dict, please put a check

if outside_quorum:
     outside_quorum = outside_quorum['outside_quorum']

Comment (anmolbabu) [inline tendrl/performance_monitoring/sds/ceph/ceph_plugin.py]: Done

**Comment by anmolbabu** [comment]
@r0h4n @nthomas-redhat @shtripat Please review

**Pull request merged**